networkx/algorithms/similarity.py

```diff
@@ -187,7 +186,7 @@ def graph_edit_distance(
 
     """
     bestcost = None
-    for vertex_path, edge_path, cost in optimize_edit_paths(
+    for _, _, cost in optimize_edit_paths(
```

Comment:
i don't necessarily think this an improvement. I would typically *prefix* unused loop variables with an underscore, but by removing the names entirely you've stripped out some of the context

```suggestion
    for _vertex_path, _edge_path, cost in optimize_edit_paths(
```

Reply:
Well this is what flake8 would complain about 🙃

Reply:
That depends on the configuration that you use. A leading underscore is a widely used convention (in many languages, not just python) for indicating an unused variable, that you still wish to explicitly name.

I'm not sure vanilla flake8 will pick this up, but `flake8-bugbear` certainly will